pkg/apis/intevents/v1beta1/pullsubscription_validation.go

```diff
@@ -33,6 +33,11 @@ import (
 
 func (current *PullSubscription) Validate(ctx context.Context) *apis.FieldError {
 	errs := current.Spec.Validate(ctx).ViaField("spec")
+
+	if apis.IsInUpdate(ctx) {
```

Comment:
were all these ones missed in the previous PR so you are adding them now?
the sources v1 and some of this?
Is that right?
If so, can you update the PR title

Reply:
Nope, the topic and pullsubscription were a part of previous PR. I had to merge https://github.com/google/knative-gcp/pull/1523 to this branch to avoid conflicts.